server/views/pages/component.njk

```diff
@@ -89,24 +91,24 @@
       {% for item in component.dependencyComparison.items %}
         {%  if (component.language == 'Kotlin') and (item.current != '-' and item.current) and item.status == 'needs-attention' or item.status == 'needs-upgrade' %}
           {%  set _ = filteredRows.push([
```

Comment:
`_` is not a descriptive name?

Also noticed a lot of double spaces after the `{%`